CharacterCreation/07_Items.txt

```diff
@@ -2,164 +2,178 @@
 DC to hack without skill points: 80
 
-Road Flare		$15:
-A phosphorus red stick with a strike pad at the end. Will put off soft red 
-light for 5m in every direction for about an hour. Burns in rain, mud, 
-even burns underwater. Don't touch the end or you'll take 10 fire damage.
+Road Flare ($15):
+    A phosphorus red stick with a strike pad at the end. Will put off soft red 
+light for 5m in every direction for about an hour. Burns in rain, mud, even 
+burns underwater. Don't touch the end or you'll take 10 fire damage.
 
-Comfy Bed Roll		$55:
-An insulated and padded bedroll. Great for missions which last more than a
+Comfy Bed Roll ($55):
+    An insulated and padded bedroll. Great for missions which last more than a
 couple of days. Not rated for temperatures above 43 degrees C or below -5 C.
 
-Walkie-Talkie		$45:
-A simple 30 channel push to talk hand held radio. Has a 2km range. Comes 
+Walkie-Talkie ($45):
+    A simple 30 channel push to talk hand held radio. Has a 2km range. Comes 
 with a handy clip to attach to a belt or combat harness. Simple encryption can
 be broken with a DC 45 hacking skill check.
 
-Radio Headset		$65:
-A slightly more sophistocated communication system. Fits well under most 
+Radio Headset ($65):
+    A slightly more sophistocated communication system. Fits well under most 
 combat helmets and is available in different sizes for most races or species.
 Uses passive filtering to reduce background noise. Capable of broadcasting
 while receiving. Better encryption means it takes a hacking check of DC 64
 to listen in. A DC check of 78 to make a call on an enemy line. Range of 3km.
 
-Integrated Remote Verbal Communication Module $80:
-Cybernetic customers such as cyborgs or Security Droids may purchase and
+Integrated Remote Verbal Communication Module ($80):
+    Cybernetic customers such as cyborgs or Security Droids may purchase and
 install an integrated radio broadcasting module. These have a range of 2km
 and a DC of 82 to hack into, since they can use the host's own encryption 
 hardware.
 
-Subdermal Mic		$230:
-The best combat communication system available on the public market. A
+Subdermal Mic ($230):
+    The best combat communication system available on the public market. A
 microphone is actually injected under the users' skin and rests next to the
 combatant's jaw bone. It uses the vibration of the jaw to send clear voice
 sounds even over cacophanous gunfire. A similar speaker is injected in the
 back of the cheek near the ear. Impossible to see with the naked eye without 
 invading the wearer's personal space. DC 140 to hack.
-
-Other misc. items are available for sale in the world. While you're playing be 
-on the lookout for shops or characters who might sell you more things. If you
-want to buy other starting equipment, ask your DM if something specific is 
-available and how much it might cost. Equipment that you start with generally
-depends on your character's fist mission. 
```

Comment:
Do we want to add a section somewhere talking about this? The malleability of the game to allow additional unlisted items?

I am of the opinion that we should list out all the items normally available and that we should leave it as implicit that the DM can make up what they want. But I offer this question just to be thorough.

Reply:
I'm really torn on whether to leave it open ended or not. Let's call it scope creep in terms of this particular PR, and open another one to tackle misc. items. I think I'm going to end up coming down on the side of not allowing unlisted items as well; that lets a dungeon creator know what items a character is likely to start with. But I think there's still room for debate. Since this is a mechanics question, the final decision will be yours, 1Source.